Refactor out woke instructions (#158)
canonical/sphinx-docs-starter-pack: readme.rst

To add exceptions for words flagged by the spelling check, edit the ``.custom_wordlist.txt`` file.
You shouldn't edit ``.wordlist.txt``, because this file is maintained and updated centrally and contains words that apply across all projects.

Customisation of inclusive language checks
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

By default, the inclusive language check is applied only to reST files located
under the documentation directory (usually ``docs``). To check Markdown files,
for example, or to use a location other than the ``docs`` sub-tree, you must
change how the ``woke`` tool is invoked from within ``docs/Makefile`` (see
the `woke User Guide <https://docs.getwoke.tech/usage/#file-globs>`_ for help).

> Some circumstances may compel you to retain some non-inclusive words. In such

Reviewer: ```suggestion
Some circumstances might require you to use some non-inclusive words. In such
```
Let's keep it simple. ;)
